paperslicer/pipeline.py

```
@@ -62,3 +84,439 @@ def process(self, pdf_path: str) -> PaperRecord:
+            try:
+                from paperslicer.grobid.figures import parse_figures_tables, parse_table_data
+                items = parse_figures_tables(tei)
+                if items:
+                    md_plus["figures_list"] = [it for it in items if it.get("type") == "figure"]
+                    md_plus["tables_list"] = [it for it in items if it.get("type") == "table"]
+                # Optional: structured table rows
+                table_rows = parse_table_data(tei)
+                if table_rows:
+                    md_plus["tables_data"] = table_rows
+            except Exception:
+                pass
```

Ruff (0.12.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)